run/test/specs/message_checks_groups_ios.spec.ts

```diff
@@ -0,0 +1,507 @@
+import { iosIt } from "../../types/sessionIt";
+import { sleepFor, clickOnXAndYCoordinates } from "./utils";
+import { newUser } from "./utils/create_account";
+import { createGroup } from "./utils/create_group";
+import {
+  SupportedPlatformsType,
+  openAppThreeDevices,
+  closeApp,
+} from "./utils/open_app";
+import { runScriptAndLog } from "./utils/utilities";
+
+async function sendImageGroup(platform: SupportedPlatformsType) {
+  const testGroupName = "Message checks for groups";
+  const ronSwansonBirthday = "196705060700.00";
+  const { device1, device2, device3 } = await openAppThreeDevices(platform);
+  // Create users A, B and C
+  const [userA, userB, userC] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+    newUser(device3, "Charlie", platform),
+  ]);
+
+  // Create contact between User A and User B
+  await createGroup(
+    platform,
+    device1,
+    userA,
+    device2,
+    userB,
+    device3,
+    userC,
+    testGroupName
+  );
+  const testMessage = "Ron Swanson doesn't like birthdays";
+  const replyMessage = `Replying to image from ${userA.userName} in ${testGroupName}`;
+  await device1.clickOnElement("Attachments button");
+  await sleepFor(100);
+  await clickOnXAndYCoordinates(device1, 34, 498);
+  const permissions = await device1.doesElementExist({
+    strategy: "accessibility id",
+    selector: "Allow Access to All Photos",
+    maxWait: 1000,
+  });
+  if (permissions) {
+    try {
+      await device1.clickOnElement(`Allow Access to All Photos`);
+      // Select video
+    } catch (e) {
+      console.log("No permissions dialog");
+    }
+  } else {
+    console.log("No permissions dialog");
+  }
+  const testImage = await device1.doesElementExist({
+    strategy: "accessibility id",
+    selector: `1967-05-05 21:00:00 +0000`,
+    maxWait: 2000,
+  });
+  if (!testImage) {
+    await runScriptAndLog(
+      `touch -a -m -t ${ronSwansonBirthday} 'run/test/specs/media/test_image.jpg'`
+    );
+
+    await runScriptAndLog(
+      `xcrun simctl addmedia ${
+        process.env.IOS_FIRST_SIMULATOR || ""
+      } 'run/test/specs/media/test_image.jpg'`,
+      true
+    );
+  }
```

Comment:
would be good to merge that push and addmedia logic for android/iOS into the device class